lib/zendesk_apps_support/validations/libraries.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module ZendeskAppsSupport
+  module Validations
+    module Libraries
+      LEGACY_ASSETS = %r{https?://\S*assets\.zendesk\.com}
+      GARDEN_CDN_URL = /<link\s.*href="(#{LEGACY_ASSETS.source}\S*zendesk_garden\.css).*>/
+      LEGACY_SDK_URL = %r{<script\s.*src="(#{LEGACY_ASSETS.source}\S*\/((\d(\.\d+){1,2})|latest)\/zaf_sdk\.js).*>}
```

Comment:
Feels slightly weird to include validations against legacy features. This code should eventually go away, right? Should we have a comment to explain why this is needed?

Reply:
Yep this should absolutely be temporary, _but_ both old assets are going to hang around for a while and it should help to deprecate the legacy assets. I'll add a comment to explain this 👍